src/test/java/com/hocs/test/glue/CreateCaseStepDefs.java

```diff
@@ -45,34 +44,28 @@ public void iAmPresentedWith(String userView) {
                 createCase.assertNoOptionsAvailable();
                 break;
             default:
-                fail(userView + " is not defined with CreateCaseStepDefs.iAmPresentedWith");
+                System.out.println(userView
+                        + " is not defined within " + getClass().getSimpleName()
+                        + " class, " + getMethodName() + " method");
+                userView = null;
+                assumeNotNull(userView);
         }
     }
 
-
-   /* public void iAmPresentedWith(String userView) {
-        switch (userView.toUpperCase()) {
-            case "NO CASE TYPES":
-                createCase.assertNoOptionsAvailable();
-                break;
-            default:
-                fail(userView + " is not defined with CreateCaseStepDefs.iAmPresentedWith");
-        }
-    }*/
-
-
-
     @Given("^I create a single case \"([^\"]*)\"$")
     public void createACaseTypeSpecificCase(String caseType) {
         switch (caseType.toUpperCase()) {
             case "DCU MIN":
                 createCase.createDCUMinSingleCase();
+                Serenity.setSessionVariable("caseType").to(caseType);
```

Comment:
if you add a static import of Serenity you dont have to include the `Serenity.` part of `Serenity.setSessionVariable`

import static net.serenitybdd.core.Serenity.setSessionVariable;